### ISSUE TITLE
Buying items from sends you back to camp instead of back to blacksmith

### ISSUE THREAD

**Issue by FurSofa** [crawsome/PyRPG_Mini#3]
Solution idea(raw):

blacksmith main menu:
> blacksmith buying function
> blacksmith exit function

**Comment by crawsome** [comment]
Good idea to make it go back to the blacksmith. We can rework the flow control to not exit the BS until the hero decides to.

**Comment by crawsome** [comment]
Good issue for several reasons! I started rewriting the flow control, and you and I realized we wanted to just write a class for it. Please complete the code skeleton I sent to you with functions for merchants, and we can try to get it to work on your fork. Inheritance and modifying code created in other files is the trick here.
